app/assets/stylesheets/views/_covid.scss

```diff
@@ -80,13 +72,27 @@
   }
 }
 
+.covid__logo-wrapper {
+  margin-bottom: govuk-spacing(7);
+
+  @include govuk-media-query($from: tablet) {
+    margin-bottom: 0;
+    position: relative;
+    height: 108px;
+  }
+}
+
 .covid__logos {
   max-width: 100%;
   height: auto;
+  display: block;
+  margin: 0 auto;
```

Comment:
These lines seem to have been added to provide centering, but the SVG on mobile seems to always be as wide as the screen, so they don't seem to do anything.